src/common/java/net/minecraftforge/gradle/common/util/JavadocAdder.java

```diff
@@ -83,27 +83,27 @@ public static String buildJavadoc(String indent, String javadoc, boolean multili
         return builder.toString().replace(indent, indent);
     }
 
-    private static List<String> wrapText(String text, int len)
+    private static List<String> wrapText(@Nullable String text, int len)
     {
         // return empty array for null text
         if (text == null)
         {
-            return new ArrayList<String>();
+            return new ArrayList<>();
         }
 
         // return text if len is zero or less
         if (len <= 0)
         {
-            return new ArrayList<String>(Arrays.asList(text));
+            return new ArrayList<>(Collections.singletonList(text));
```

Comment:
Some background: this change was done during a cleanup pass, and what prompted the change was the specificity of the generics for `ArrayList`. That also prompted me to change it to a `singletonList` instead of `Arrays.asList`, since that would be more 'performant' (in the sense that `Arrays.asList` is geared towards handling arrays, while we are just passing in one value so a singleton list would be better).

While it is possible to use `ImmutableList<T>.of(T)` here instead of `Collections.<T>singletonList(T)`, my own preferrence would be using the Java-provided `Collections` method. If you are instead asking about why we are returning an `ArrayList`, it's to preserve the behavior of the method (returning a mutatable collection, whereas the three methods aforementioned do not guarantee mutability/guarnatee immutability).

If you wish, I can change it to use `ImmutableList`, but that's a negligible change with zero impact.

Reply:
I meant `Lists.newArrayList`, not necessarily immutable collections.

Reply:
I was not fully aware that the rest of the codebase used that method, as explained what I did was a sweep of the class to remove unnecessary warnings which were the uses of `<String>` instead of `<>`. You'd have to ask the creator of the class as to why they chose to create the array list directly instead of using `Lists.newArrayList`.